### ISSUE TITLE
Limit Diags not Finding Imports

### ISSUE THREAD

**Issue by mfeit-internet2** [perfsonar/pscheduler#691]
The `limit-diags` command doesn't find the files in `/usr/lib/jq/pscheduler` for import:

```
% pscheduler limit-diags --hints '{ "requester": "9.9.9.9", "server": "8.8.8.8" }' --config ./pscheduler_limits.conf ~/hole/task
Unable to read limit configuration: jq: error: module not found: pscheduler/iso8601

jq: 1 compile error
```

The limit processor in the API server works just fine.

**Comment by mfeit-internet2** [comment]
Andy says he's seeing this in the REST API, too.  Probably need to check and see if the author took 100% of what we submitted for patches.

**Comment by mfeit-internet2** [comment]
The patch to 2.2.0 added facilities to have a search path but didn't set any defaults, which is what I submitted.  Ours has been patched to set one.

**Comment by mfeit-internet2** [comment]
Reopening.  Andy's seeing it again in 4.2.0 with the stock limits file.

**Comment by mfeit-internet2** [comment]
It looks like the code to auto-find the libraries that was in the patch shipped with 4.1.6 didn't make it into 4.2.0.  I think instead of trying to retrofit it into PyJQ and introduce something that might be system-specific, I'm going to shift it into our own code.
